core/ui/EditTemplate/body/default.tid

```diff
@@ -1,18 +1,18 @@
 title: $:/core/ui/EditTemplate/body/default
 
-\function get.edit-preview-state()
+\function edit-preview-state()
```

Comment:
functions need a dot `.` in the function name. Otherwise thy can only be used with `[function[get-preview-state]]` -- We need the possibility to use it with `[tf.get-preview-state<something>]` too.

That's why I did start with `tf.function-name-with-hyphens` for TW global functions. If you search the source code for `tf.`, you will see some of them in the TW core code already. 

At the moment they are only used for `$:/tags/Global` functions.

Reply:
At the **draft pr branch** https://github.com/pmario/TiddlyWiki5/blob/toc-v5.3.x-rewrite/core/wiki/macros/toc.tid you can see it used extensively. Otherwise the code would be unreadable.